src/hooks/data/useVaultApy.ts

```diff
@@ -22,7 +40,24 @@ export function useVaultBaseApr({
 }: VaultDynamicProps) {
   // Preferred: API request
   const apiQuery = useApiVaultDynamic({ poolId, type })
-  // TODO: Fallbacks?
+
+  const vaultAprFallback = useQuery([_address, "vaultAprFallback"], {
```

Comment:
You should include `chainId` as the first item in the queryKey to be sure that we're updating data when the active chain changes. It might not be 100% needed here because the address may be unique, but its worth doing just to be certain.